plugins/micromega/g_zify.mlg

```diff
@@ -40,6 +40,8 @@ END
 TACTIC EXTEND TRANS
 | [ "zify_op"  ] -> { Zify.zify_tac }
 | [ "saturate"  ] -> { Zify.saturate }
+| [ "assert_inj" constr(t)] -> { Zify.assert_inj t }
```

Comment:
This tactic is not used ML-side, so you can remove it. It is better not to export unused tactics, as the Coq API doesn't make them internal.

Reply:
done.